### ISSUE TITLE
flyway V2.8.0 migration fails on sqlserver

### ISSUE THREAD

**Issue by scivm** [OHDSI/WebAPI#1688]
flyway V2.8.0 migration fails on sqlserver with latest code.  This is new install.

```
Migration V2.8.0.20201103171300__add_source_daimon_unique_constraint.sql failed
-------------------------------------------------------------------------------
SQL State  : S0001
Error Code : 102
Message    : Incorrect syntax near 'sd1'.
Location   : db/migration/sqlserver/V2.8.0.20201103171300__add_source_daimon_unique_constraint.sql (/var/lib/ohdsi/webapi/WEB-INF/classes/db/migration/sqlserver/V2.8.0.20201103171300__add_source_daimon_unique_constraint.sql)
Line       : 1
Statement  : delete from        OHDSI.source_daimon sd1
where sd1.priority = -1
  and sd1.source_daimon_id < (
    select max(source_daimon_id)
    from OHDSI.source_daimon sd2
    where sd1.source_id = sd2.source_id
      and sd1.daimon_type = sd2.daimon_type);

ALTER TABLE OHDSI.source_daimon ADD CONSTRAINT un_source_daimon UNIQUE (source_id,daimon_type);
```

**Comment by wivern** [comment]
@scivm Could you please ensure you are using the latest code? 
I believe the same issue was fixed by the #1685

**Comment by scivm** [comment]
ah ok.  I didn't notice the update since I cloned the repo.
